src/api/app/views/webui/user/tasks.html.haml

```diff
@@ -0,0 +1,79 @@
+- @pagetitle = "Tasks for #{@displayed_user}"
```

Comment:
This one is not necessary anymore, right?

Reply:
It is. Makes this:

![screenshot from 2018-03-14 14-08-39](https://user-images.githubusercontent.com/514785/37404131-399778e6-2791-11e8-8c80-7e596f0e31f3.png)